openghg/store/_boundary_conditions.py

```diff
@@ -76,6 +81,7 @@ def read_file(
               - a model name such as "MOZART" or "CAMS"
               - a description such as "UniformAGAGE" (uniform values based on AGAGE average)
             domain: Region for boundary conditions
+            source_format: "boundary_conditions is default"
```

Comment:
As per other comments, this should be "openghg". Would be good to include some more description here of what this is (even if copied / derived from other standardise functions).